slick/src/main/scala/slick/compiler/ForceOuterBinds.scala

```diff
@@ -54,3 +61,17 @@ class ForceOuterBinds extends Phase {
     case _ => nowrap(n)
   }
 }
+
+class ResolveHeadOption extends Phase {
```

Comment:
I dont know if that phase will stay. Its a placebo for now. Once I get the thing completed with all the types resolving completely, I will be more sure on that! This phase is not in use in the updated code.